src/main/java/uk/gov/hmcts/reform/coh/controller/onlinehearing/OnlineHearingMapper.java

```diff
@@ -15,5 +15,14 @@ public static void map(OnlineHearingResponse response, OnlineHearing onlineHeari
                 .stream()
                 .map( p -> new OnlineHearingResponse.PanelMember(p.getFullName()))
                 .collect(Collectors.toList()));
+        response.getCurrentState().setName(onlineHearing.getOnlineHearingState().getState());
+
+        if (!onlineHearing.getOnlineHearingStateHistories().isEmpty()){
+            response.getCurrentState().setDatetime
+                    (onlineHearing.getOnlineHearingStateHistories().stream().sorted(
+                            (a, b) -> (a.getDateOccurred().compareTo(b.getDateOccurred()))).collect(Collectors.toList()
+                    ).get(onlineHearing.getOnlineHearingStateHistories().size()-1).getDateOccurred().toString());
+        }
+
```

Comment:
Needs tests:
1. History is sorted correctly
2. No history does not fail
3. History is returned in the GET response
3. Multiple events can be returned in the GET response